python/pyspark/pandas/series.py

```diff
@@ -944,6 +944,57 @@ def between(self, left: Any, right: Any, inclusive: bool = True) -> "Series":
 
         return lmask & rmask
 
+    def cov(self, other: "Series", min_periods: Optional[int] = None) -> float:
+        """
+        Compute covariance with Series, excluding missing values.
+        Parameters
+        ----------
+        other : Series
+            Series with which to compute the covariance.
+        min_periods : int, optional
+            Minimum number of observations needed to have a valid result.
+
+        Returns
+        -------
+        float
+            Covariance between Series and other
+
+        Examples
+        --------
+        >>> from pyspark.pandas.config import set_option, reset_option
+        >>> set_option("compute.ops_on_diff_frames", True)
+        >>> s1 = ps.Series([0.90010907, 0.13484424, 0.62036035])
+        >>> s2 = ps.Series([0.12528585, 0.26962463, 0.51111198])
+        >>> s1.cov(s2)
+        -0.016857626527158744
+        >>> reset_option("compute.ops_on_diff_frames")
+        """
+
+        if min_periods is None:
+            min_periods = 1
```

Comment:
Just curious, can an optional int argument not have a default value where it's declared?

Reply:
For case we pass None like pandas behavior
```
>>> import pandas as pd
>>> s1 = pd.Series([0.90010907, 0.13484424, 0.62036035])
>>> s2 = pd.Series([0.12528585, 0.26962463, 0.51111198])
>>> s1.cov(s2, None)
-0.01685762652715874
```

Reply:
I know, just wondering if you can express the default in the signature? Just a python question for my own info. I understand what this does, that's not the question

Reply:
Yes, I can. I pushed a commit to change it

Reply:
@dgd-contributor I think what @srowen commented was just a question about Python.

Actually, we should follow the [pandas' API docs](https://pandas.pydata.org/docs/reference/api/pandas.Series.cov.html) what you did before :-)